data/brands/shop/supermarket.json

```diff
@@ -4745,6 +4745,17 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "Metro Supermarket"
```

Comment:
```suggestion
      "displayName": "Metro Supermarket",
```

Reply:
btw our build script uses a [JSON5](https://json5.org/) parser, but it will still output standard JSON, so `npm run build` will quietly fix small stuff like this 👍